### PR TITLE
ADFS fix

### PR DESCRIPTION
**Pull request by Joerger**

### CODE REVIEW

File: lib/services/saml.go [hunk 1]
```diff
@@ -181,7 +181,8 @@ func GetSAMLServiceProvider(sc SAMLConnector, clock clockwork.Clock) (*saml2.SAM
 	}
 
 	// adfs specific settings
-	if sc.GetAudience() == teleport.ADFS {
+	if sc.GetProvider() == teleport.ADFS {
+		log.Debug("[SAML] setting ADFS values")
```
Comment (russjones) [inline lib/services/saml.go:185]: Let's use component logging here.

**Review by russjones — COMMENTED** [review]
@Joerger Did you test this with @stevenGravy ?

File: lib/services/saml.go (hunk 1 above)
Comment (Joerger) [reply lib/services/saml.go]: He said he'd let me know when he tests it. It was a very simple typo, so I'm confident that this will work now.

**Comment by Joerger** [comment]
> @Joerger Did you test this with @stevenGravy ?

He said he'd let me know when he tests it. It was a very simple typo, so I'm confident that this will work now.

File: lib/services/saml.go (hunk 1 above)
Comment (Joerger) [reply lib/services/saml.go]: Did I add that correctly?

**Review by russjones — APPROVED** [review]
@Joerger Approved pending @stevenGravy confirming.

File: lib/services/saml.go [hunk 2]
```diff
@@ -181,7 +181,10 @@ func GetSAMLServiceProvider(sc SAMLConnector, clock clockwork.Clock) (*saml2.SAM
 	}
 
 	// adfs specific settings
-	if sc.GetAudience() == teleport.ADFS {
+	if sc.GetProvider() == teleport.ADFS {
+		log.WithFields(log.Fields{
+			trace.Component: teleport.ComponentSAML,
+		}).Debug("setting ADFS values")
```
Comment (r0mant) [inline lib/services/saml.go:187]: Nit: `Setting ADFS values.` (capitalize and dot at the end).

**Review by r0mant — APPROVED** [review]

**Pull request merged**